engine/src/main/java/org/terasology/rendering/dag/stateChanges/BindFBO.java

```diff
@@ -50,6 +50,11 @@ public StateChange getDefaultInstance() {
         return defaultInstance;
     }
 
+
+    public static void setDefaultInstance(BindFBO defaultInstance) {
```

Comment:
In line with what written below, I don't think we need a setDefaultInstance method.